rest-api/main.py

```diff
@@ -0,0 +1,1824 @@
+#!/usr/bin/env python
```

Comment:
Is it an option to rename this file from main.py to something that indicate its purpose? Like bareos-rest-api.py. Otherwise it may confusing in the process list when running.

Reply:
renamed to bareos-restapi.py